core/storage/trie/serialization/trie_serializer_impl.cpp

```diff
@@ -75,11 +84,8 @@ namespace kagome::storage::trie {
                                               // the DB separately
                 }
               }
-              if (auto value_data = std::get_if<Codec::ValueData>(&visitee);
-                  value_data != nullptr) {
-                return batch->put(value_data->hash, value_data->value.view());
-              }
-              BOOST_UNREACHABLE_RETURN();
+              auto value_data = std::get<Codec::ValueData>(visitee);
```

Comment:
Should we make copy there? 
```suggestion
              auto& value_data = std::get<Codec::ValueData>(visitee);
```